widget/embedded/src/utils/swap.ts

```diff
@@ -1,26 +1,31 @@
-import BigNumber from 'bignumber.js';
-import {
+/* eslint-disable @typescript-eslint/no-magic-numbers */
```

Comment:
?

Reply:
We have an ESLint rule against using hard-coded numbers in our code. This file contains some hard-coded numbers, which I suggest addressing at a later time because it's not related to this PR.